### PR TITLE
CIV-9364 making camunda changes for Demages Multiclaims

### PR DESCRIPTION
**Pull request by indu2019**
Change Camunda flow diagram for Multi track claims

Context 
During the Analysis of [CIV-8602](https://tools.hmcts.net/jira/browse/CIV-8602), it was discovered that the Camunda flow diagram for Multi-track claims has changed.

Prior to SDO, multi track claims were NOT moved to Judicial Referral and were going to Notify RPA on case handed offline.

After SDO, multi track claims are now moving to Judicial Referral and are going to Notify RPA on Continuous feed.

We believe that because multi track claims are now moving to Notify RPA on Continuous feed, this is causing them not to be picked up by the batch report.

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-9364

### Change description ###
Change Camunda flow diagram for Multi track claims

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

**Review by asthamalviya — DISMISSED** [review]

File: src/main/java/uk/gov/hmcts/reform/civil/service/flowstate/FlowFlag.java [hunk 1]
```diff
@@ -8,6 +8,7 @@ public enum FlowFlag {
     PIP_ENABLED,
     NOTICE_OF_CHANGE,
     AGREED_TO_MEDIATION,
+    MULTI_CLAIM_YES,
```
Comment (HarryH96) [inline src/main/java/uk/gov/hmcts/reform/civil/service/flowstate/FlowFlag.java:11]: IS_MULTI_TRACK ?

File: src/main/java/uk/gov/hmcts/reform/civil/service/flowstate/FlowPredicate.java [hunk 2]
```diff
@@ -621,6 +622,10 @@ public static final boolean getPredicateTakenOfflineByStaffAfterNotificationAckn
             && caseData.getApplicant1ResponseDeadline().isBefore(LocalDateTime.now())
             && caseData.getApplicant1ProceedWithClaim() == null;
 
+    public static final Predicate<CaseData> demageMultiClaim = caseData ->
+        AllocatedTrack.MULTI_CLAIM.equals(caseData.getAllocatedTrack())
+        && CaseCategory.UNSPEC_CLAIM.equals(caseData.getCaseAccessCategory());
```
Comment (HarryH96) [inline src/main/java/uk/gov/hmcts/reform/civil/service/flowstate/FlowPredicate.java:627]: Does this not also impact SPEC Claims too? 
Claim Type should not matter - all Multi Track should drop offline after issue

File: src/main/java/uk/gov/hmcts/reform/civil/service/flowstate/FlowFlag.java (hunk 1 above)
Comment (indu2019) [reply src/main/java/uk/gov/hmcts/reform/civil/service/flowstate/FlowFlag.java:11]: @HarryH96 are you suggesting that I should rename this field ?

File: src/main/java/uk/gov/hmcts/reform/civil/service/flowstate/FlowPredicate.java (hunk 2 above)
Comment (indu2019) [reply src/main/java/uk/gov/hmcts/reform/civil/service/flowstate/FlowPredicate.java:627]: @HarryH96 at this ticket is for Demages thats why I am setting MULTI_CLAIM_YES only if claim is demages and Multiclaim.

**Review by HarryH96 — CHANGES_REQUESTED** [review]
Unit Tests added / updated?

File: src/main/java/uk/gov/hmcts/reform/civil/service/flowstate/FlowFlag.java (hunk 1 above)
Comment (indu2019) [reply src/main/java/uk/gov/hmcts/reform/civil/service/flowstate/FlowFlag.java]: variable renamed and unit test added.

**Review by HarryH96 — DISMISSED** [review]

**Review by asthamalviya — APPROVED** [review]

**Review by asthamalviya — APPROVED** [review]

**Review by HarryH96 — APPROVED** [review]

**Review by sabahirfan — APPROVED** [review]

**Pull request merged**